mono/mini/debugger-agent.c

```diff
@@ -847,18 +847,18 @@ parse_address (char *address, char **host, int *port)
 static void
 print_usage (void)
 {
-	fprintf (stderr, "Usage: mono --debugger-agent=[<option>=<value>,...] ...\n");
-	fprintf (stderr, "Available options:\n");
-	fprintf (stderr, "  transport=<transport>\t\tTransport to use for connecting to the debugger (mandatory, possible values: 'dt_socket')\n");
-	fprintf (stderr, "  address=<hostname>:<port>\tAddress to connect to (mandatory)\n");
-	fprintf (stderr, "  loglevel=<n>\t\t\tLog level (defaults to 0)\n");
-	fprintf (stderr, "  logfile=<file>\t\tFile to log to (defaults to stdout)\n");
-	fprintf (stderr, "  suspend=y/n\t\t\tWhether to suspend after startup.\n");
-	fprintf (stderr, "  timeout=<n>\t\t\tTimeout for connecting in milliseconds.\n");
-	fprintf (stderr, "  server=y/n\t\t\tWhether to listen for a client connection.\n");
-	fprintf (stderr, "  keepalive=<n>\t\t\tSend keepalive events every n milliseconds.\n");
-	fprintf (stderr, "  setpgid=y/n\t\t\tWhether to call setpid(0, 0) after startup.\n");
-	fprintf (stderr, "  help\t\t\t\tPrint this help.\n");
+	g_print ("Usage: mono --debugger-agent=[<option>=<value>,...] ...\n");
+	g_print ("Available options:\n");
+	g_print ("  transport=<transport>\t\tTransport to use for connecting to the debugger (mandatory, possible values: 'dt_socket')\n");
+	g_print ("  address=<hostname>:<port>\tAddress to connect to (mandatory)\n");
+	g_print ("  loglevel=<n>\t\t\tLog level (defaults to 0)\n");
+	g_print ("  logfile=<file>\t\tFile to log to (defaults to stdout)\n");
+	g_print ("  suspend=y/n\t\t\tWhether to suspend after startup.\n");
+	g_print ("  timeout=<n>\t\t\tTimeout for connecting in milliseconds.\n");
+	g_print ("  server=y/n\t\t\tWhether to listen for a client connection.\n");
+	g_print ("  keepalive=<n>\t\t\tSend keepalive events every n milliseconds.\n");
+	g_print ("  setpgid=y/n\t\t\tWhether to call setpid(0, 0) after startup.\n");
+	g_print ("  help\t\t\t\tPrint this help.\n");
```

Comment:
`g_printerr ()` is the equivalent to `fprintf (stderr, ...)`

Reply:
+1